extras/mobile-view.rst

```diff
@@ -0,0 +1,156 @@
+
+          Ticket Create Screen
+
+   * - .. figure:: /images/extras/mobile-view/mobile-view-ticket-detail-view.png
+          :alt: Mobile View Ticket Detail View
+          :align: center
+          :width: 100%
+
+          Ticket Detail View
+
+     - .. figure:: /images/extras/mobile-view/mobile-view-article-reply.png
+          :alt: Mobile View Article Reply Dialog
+          :align: center
+          :width: 100%
+
+          Article Reply Dialog
+
+     - .. figure:: /images/extras/mobile-view/mobile-view-organization.png
+          :alt: Mobile View Ticket Organization Tab
+          :align: center
+          :width: 100%
+
+          Ticket Organization Tab
+
+.. hint::
+
+   We intentionally do not provide specific instructions and comprehensive
+   documentation for the mobile view! The overall UX should be intuitive and
+   self-explanatory in most cases.
+
+Features
+--------
+
+Mobile view provides you with a way to do your common Zammad daily tasks while
+on the go:
+
+   * Manage & use your ticket overviews
+   * Search for existing records
+   * Create a new ticket
+   * Reply in an already existing ticket
+   * Modify ticket attributes
+   * Modify customer attributes
+   * Modify organization attributes
+
+Mobile view also has some exclusive features:
+
+   * Innovative Zammad UX
+   * Next-level accessibility
+   * Ready for multi-device usage
+   * Progressive web app (PWA) support
+
+Limitations
+-----------
+
+Mobile view is also currently missing some features which are provided by the
+desktop view:
+
+   * Ticket Article Time Accounting
+   * Ticket Article **Split** Action
+   * Linked Tickets & Ticket **Link** Action
+   * Ticket Macros
+   * Ticket History
+   * Ticket Create Templates & Shared Drafts
+
+Additionally, certain features were intentionally omitted in order to improve
+the focus on important information:
+
+   * Most Management Features (except ticket user and organization management)
+   * Most Knowledge Base Features (except ticket integration)
+   * Most User Profile Functions (except avatar and language preferences)
+   * Reports
+   * Caller Log
+   * Live Chat
+
+Access
+------
+
+Zammad now implements a mobile device detection, which results in automatic
+redirection to mobile view. Even with this mechanism in place it's possible to
+explicitly switch between the views by using app links:
+
+   * **Continue to desktop** link in mobile view
+
+     .. container:: cfloat-left
+
+        .. figure:: /images/extras/mobile-view/mobile-view-login-continue-to-desktop.png
+           :alt: Continue to Desktop Link on Login Screen
+           :align: center
+           :width: 85%
+
+           Login Screen
+
+     .. container:: cfloat-right
+
+        .. figure:: /images/extras/mobile-view/mobile-view-account-continue-to-desktop.png
+           :alt: Continue to Desktop Link in User Account Overview
+           :align: center
+           :width: 85%
+
+           User Account Overview
+
```

Comment:
Floats should be cleared to ensure that following directives are not broken.
You can do so by using 

```
     .. container:: cfloat-clear

        x
```

This also affects line 154